### ISSUE TITLE
Emote Modifiers and Deskmods Lack Explanation

### ISSUE THREAD

**Issue by Crystalwarrior** [Salanto/AO2-CharIniMaker#2]
Prompted by this conversation:
![image](https://user-images.githubusercontent.com/3470436/197624740-ec661d68-1176-4850-90f9-413aa28b0bb0.png)

![image](https://user-images.githubusercontent.com/3470436/197623666-edb00f58-2eb4-4e44-aa3d-70ae84982f14.png)

Explanations can come in tl;dr form from here: https://github.com/AttorneyOnline/docs/blob/master/docs/development/network.md#in-character-message

Relevant info from the docs (with slight corrections from memory, signified by stricken text):

- **desk_mod**: Whether or not to override desk appearance.
  * `-1`/`chat`: Positions "def", "pro", and "wit" default to desk and the positions "hld", "hlp" and "jud" to no desk.
  * `0`: desk is hidden
  * `1`: desk is shown
  * `2`: desk is hidden during preanim, shown when it ends
  * `3`: desk is shown during preanim, hidden when it ends
  * `4`: desk is hidden during preanim, character is centered and pairing is ignored, when it ends desk is shown and pairing is restored
  * `5`: desk is shown during preanim, when it ends character is centered and pairing is ignored

- **emote_modifier**: A number that dictates emote behavior:
  * `0`: do not play preanimation; overridden to 2 by a non-`0` objection modifier
  * `1`: play preanimation (and sfx)
  * `2`: play preanimation and play objection
  * `3`: _unused_
  * `4`: _unused_
  * `5`: no preanimation and zoom
  * `6`: ~~objection and zoom, no preanim~~ preanim and zoom

**So, to condense it into the dropdown:**

modifier:
 * `0` idle
 * `1` preanim
 * `5` zoom
 * `6` preanim zoom

deskmod:
 * `-1` benches
 * `0` no desk
 * `1` show desk
 * `2` show desk only on idle
 * `3` show desk only on preanim
 * `4` no desk and no pair only on preanim
 * `5` show desk and no pair

**Comment by Salanto** [comment]
Yeah- While this is super convenient, Qt really fucking hates it.
Will need to do some annoying hardcoded table information to make this work.

**Comment by Salanto** [comment]
If we drop chat I will add it literally this night.

**Comment by Crystalwarrior** [comment]
done lol
